src/codeu/chat/server/Server.java

```diff
@@ -176,6 +177,19 @@ public void onMessage(InputStream in, OutputStream out) throws IOException {
       }
     });
 
+       // New Interest - A client wants to add a new interest to the back end.
```

Comment:
Also minor, but I would put this with the other creation methods like newUser and newConversation.